docs/source/_json/history_get.resp

```diff
@@ -0,0 +1,36 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "action": null, 
+    "actor": {
+      "@id": "http://localhost:55001/plone/@users/test_user_1_", 
+      "fullname": "", 
+      "id": "test_user_1_", 
+      "username": "test-user"
+    }, 
+    "comments": "", 
+    "review_state": "private", 
+    "state_title": "Private", 
+    "time": "2016-10-21T19:00:00", 
+    "transition_title": "Create", 
+    "type": "workflow"
+  }, 
+  {
+    "action": "Edited", 
+    "actor": {
+      "@id": "http://localhost:55001/plone/@users/test-user", 
+      "fullname": "test-user", 
+      "id": "test-user", 
+      "username": null
+    }, 
+    "comments": null, 
+    "content_url": "http://localhost:55001/plone/front-page/?version_id=0", 
```

Comment:
Excuse my ignorance about how CMFEditions work. Why does the first entry does not provide a "content_url" and it does not start with version 0? I guess this exposes how Plone works internally (or used to work with AT). The object is always created empty first and then "edited" for the first time when the user submits the add form? If this is the case one could argue that this is an implementation detail that an API should hide.